packages/calypso-e2e/src/lib/pages/editor-page.ts

```diff
@@ -204,6 +204,30 @@ export class EditorPage {
 
 	//#region Basic Entry
 
+	/**
+	 * Loads HTML into the editor.
+	 * @param content Content to load.
+	 */
+	async loadHtmlContent( content: string ) {
+		const editorParent = await this.getEditorParent();
+		await editorParent.evaluate( ( _, html ) => {
+			// @ts-expect-error Untyped global variable.
+			const { parse } = window.wp.blocks;
+			// @ts-expect-error Untyped global variable.
+			const { dispatch } = window.wp.data;
+			const blocks = parse( html );
+
+			blocks.forEach( ( block: any ) => {
+				if ( block.name === 'core/image' ) {
+					delete block.attributes.id;
+					delete block.attributes.url;
```

Comment:
Do we do this in GB?

Reply:
yes https://github.com/WordPress/gutenberg/blob/79234bd5b182a8be79129cb26db6b76a63daac0d/test/performance/fixtures/perf-utils.ts#L147